Public/Src/Cache/ContentStore/Distributed/NuCache/TransitioningContentLocationStore.cs

```diff
@@ -252,6 +252,17 @@ public Result<MachineLocation> GetRandomMachineLocation(IReadOnlyList<MachineLoc
             return _localLocationStore.ClusterState.GetRandomMachineLocation(except);
         }
 
+        /// <inheritdoc />
+        public Result<MachineLocation[]> GetDesignatedLocations(ContentHash hash)
+        {
+            if (_configuration.HasReadMode(ContentLocationMode.LocalLocationStore))

```

Comment:
Minor: This organization breaks both conventions for if/else. Convention 1:
  if (exception) { do exceptional thing; }
  do normal thing;
Convention 2:
  if (exception) {
    do exceptional thing;
  } else {
    do normal thing;
  }
But nobody does:
  if (!exception) { do normal thing; }
  do exceptional thing;